openssl/README.md

```
@@ -9,9 +9,13 @@ docker build -t openssl:tagname -f Dockerfile .
 ```
 
 ## Generate via docker
-* Create a volume that points to the directory local `/etc/ssl` by using the command provided below.
+* Create a volume that points to a dedicated, empty host directory (do
+  **not** point it at the host's `/etc/ssl` — that lets the container
+  overwrite real system certificates, keys, or trust store).
   ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)